walletlibrary/src/main/java/com/microsoft/walletlibrary/util/Constants.kt

```diff
@@ -5,6 +5,8 @@ object Constants {
     const val PURE_ISSUANCE_FLOW_VALUE = "create"
     const val IDTOKENHINT_CONFIGURATION = "https://self-issued.me"
 
+    // Feature flags for Access Token flow preview feature.
     const val OPENID4VCI_ACCESS_TOKEN = "OpenID4VCIAccessToken"
```

Comment:
maybe add Feature Flag or Preview Feature to variable name and string to ensure we know it is for a feature flag.